src/main/java/org/myrobotlab/framework/MethodCache.java

```diff
@@ -113,7 +115,7 @@ public static MethodCache getInstance() {
       if (instance == null) {
         instance = new MethodCache();
         instance.excludeMethods.add("main");
-        instance.excludeMethods.add("getMetaData");
+        instance.excludeMethods.add("getMetaData"); // why ?

```

Comment:
not sure why.. don't understand why we wouldn't want folks to be able to invoke getMetaData ...  seems like we can remove this exclude

Reply:
agreed ... removed from exclusion